### ISSUE TITLE
Kong 0.14.1 - Runtime Error

### ISSUE THREAD

**Issue by zeeshen** [Kong/kong-plugin-prometheus#27]
Enable prometheus plugin globally cause err

```
tail -f error.log
2018/10/17 08:54:51 [error] 1397#0: *267915 lua entry thread aborted: runtime error: ...local/share/lua/5.1/kong/plugins/prometheus/exporter.lua:66: attempt to index field 'service' (a nil value)
stack traceback:
coroutine 0:
	...local/share/lua/5.1/kong/plugins/prometheus/exporter.lua: in function 'log'
	.../local/share/lua/5.1/kong/plugins/prometheus/handler.lua:16: in function <.../local/share/lua/5.1/kong/plugins/prometheus/handler.lua:11>, context: ngx.timer, client: xxxx, server: 0.0.0.0:80
```

And no API data
```
curl localhost:8001/apis
{"total":0,"data":[]}
```

kong version 0.14.1

**Comment by hbagdi** [comment]
Could you list the plugins being executed?
Is there a request termination or some sort plugin being executed?
I'll fix the issue in upcoming 0.3.1.

**Comment by hbagdi** [comment]
This happens if there is an unmatched route and a 404 from Kong.
In Kong 0.14.0, global log plugins will be executed for an unmatched route. 
This should be harmless but will be fixed.

**Comment by hbagdi** [comment]
This should be fixed with #28 and is available with 0.3.2 version of the plugin, which will be bundled with the next release of Kong.
